srcpkgs/simutron/patches/remove-bitbang-from-firmware-c.patch

```diff
@@ -0,0 +1,13 @@
+--- plugins/avrmcu/plugin/firmware.c
```

Comment:
please state a reason for this patch

Reply:
please state the reason for this patch in the patch file, nobody reads github conversations